### ISSUE TITLE
Eosinophilic neuronal inclusion bodies

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10517]
**Preferred term label:**
Eosinophilic neuronal inclusion bodies

**Synonyms**
Collins bodies; periodic acid-Schiff (PAS) positive but diastase resistant, composed of neuroserpin (PI12)

**Definition (free text, please give PubMed ID)**
https://omim.org/entry/604218

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0100314

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
familial encephalopathy with neuroserpin inclusion bodies

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001062
- label: Eosinophilic neuronal inclusion bodies
- parents: [Abnormal neuron morphology (HP:0012757)](https://hpo.jax.org/app/browse/term/HP:0012757)
- pmid: PMID:11138927
- def: The presence of intracellular inclusion bodies (aggregates of stainable substances, usually proteins) in neurons that stain with eosin, a red or pink dye.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10517
- synonyms:
